tests/test_listeners_kinto_changes.py

```diff
@@ -148,10 +148,11 @@ def test_kcl_posts_on_matching_records(match_buckets_a_resource):
         {'new': changes_record('a', 'c')},
     ]
     request = DummyRequest()
+    request.registry.storage.resource_timestamp.return_value = 42
     event = events.ResourceChanged(PAYLOAD, single_record, request)
 
     listener(event)
-    client.send_version.assert_called_with('broadcaster', 'monitor_changes', '"123"')
+    client.send_version.assert_called_with('broadcaster', 'monitor_changes', '"42"')
```

Comment:
Maybe we should check (at least once) that the args to `resource_timestamp` are correct?